modin/pandas/test/test_io.py

```diff
@@ -863,7 +903,10 @@ def test_read_csv_s3(self):
     @pytest.mark.parametrize("names", [list("XYZ"), None])
     @pytest.mark.parametrize("skiprows", [1, 2, 3, 4, None])
     def test_read_csv_skiprows_names(self, names, skiprows):
-
+        if Backend.get() == "Omnisci" and names is None and skiprows in [1, None]:
+            pytest.xfail(
+                "processing of duplicated columns in OmniSci backend is not supported yet - issue #3080"
```

Comment:
the message seems off - why it talks about duplicated columns and you check `skiprows`?

Reply:
Indeed this test is `xfailed` because of described reason (https://github.com/modin-project/modin/pull/2905#discussion_r637877703).